### PR TITLE
feat: add OFREP and update dependencies

### PR DESCRIPTION
**Pull request by lukas-reining**
## This PR

- Adds OFREP to playground and updates packages that are straight forward to update

### CODE REVIEW

File: packages/ui/src/app/demos.tsx [hunk 1]
```diff
@@ -73,6 +78,30 @@ export class Demos extends Component<
...
       },
     },
+    [FLAGD_OFREP_PROVIDER_ID]: {
+      factory: () => {
+        const ofrepConfig = this.state.availableProviders.find((p) => p.id === FLAGD_OFREP_PROVIDER_ID);
+        const tls = ofrepConfig?.tls ?? false;
+        const host = ofrepConfig?.host ?? 'localhost';
+        const port = ofrepConfig?.port ?? 8016;
+        const baseUrl = `${tls ? 'https' : 'http'}://${host}:${port}`;
+        return new OFREPWebProvider({ baseUrl }, console);
```
Comment (beeme1mr) [inline packages/ui/src/app/demos.tsx:88]:
```suggestion
        return new OFREPWebProvider({ baseUrl, pollInterval: 1000 }, console);
```

I think we should set the polling interval to be more aggressive. This is for demo purposes, so waiting up to 30 seconds could lead to awkward silence 😄.

**Review by beeme1mr — APPROVED** [review]
This works great! It's really cool to see OFREP plug into the demo so cleanly.

File: packages/ui/src/app/demos.tsx (hunk 1 above)
Comment (lukas-reining) [reply packages/ui/src/app/demos.tsx:88]: Should we go to something as little as 10 seconds?

File: docker-compose.yaml [hunk 2]
```diff
@@ -118,13 +119,14 @@ services:
       - ./config/flagd/flags.json:/etc/flagd/flags.json
     ports:
       - '8013:8013'
+      - '8016:8016'
 
   go-feature-flag:
-    image: thomaspoignant/go-feature-flag-relay-proxy:v1.21.0
+    image: thomaspoignant/go-feature-flag:v1.25.0
```
Comment (thomaspoignant) [inline docker-compose.yaml:125]:
```suggestion
    image: thomaspoignant/go-feature-flag:v1.25.1
```

**Review by thomaspoignant — APPROVED** [review]
As you mentioned to me, there is a small issue in OFREP in `v1.25.0`, the next version is fixing it (`v1.25.1`).

**Pull request merged**